UltraStar Play/Assets/Common/UI/UIUtils.cs

```diff
@@ -0,0 +1,91 @@
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UIElements;
+
+public static class UIUtils
+{
+    // Returns a nicely readable string from a filename, e.g. "theme_blue" will become "Theme Blue"
+    public static string BeautifyString(string input)
+    {
+        input = input.Replace("_", " ").Replace("-", " ");
+        char[] chars = input.ToCharArray();
+        bool lastWasSpace = true;
+        for (int c = 0; c < chars.Length; c++)
+        {
+            if (lastWasSpace) chars[c] = char.ToUpperInvariant(input[c]);
+            lastWasSpace = char.IsWhiteSpace(chars[c]);
+        }
+        return new string(chars);
+    }
+
+    public static void ForEachElementWithClass(VisualElement root, Action<VisualElement> callback, params string[] classNames)
+    {
+        foreach (string className in classNames)
+        {
+            root.Query(null, className).ForEach(callback);
+        }
+    }
+
+    static readonly Dictionary<VisualElement, Tuple<Color, Color>> ElementsBgColorsDict = new ();
```

Comment:
- make it a property or start with a lower case letter.
    -  `Name 'ElementsBgColorsDict' does not match rule 'static_fields_should_be_camel_case'. Suggested name is 'elementsBgColorsDict'.`